src/hooks/useDropdownOptions.ts

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export interface Item {
```

Comment:
I used to only use interface. Now it seems there aren't many worthwhile benefits to using interface over type. 

When should we use one over the other? Or, which should we use, interface or type?

Reply:
That's something I've struggled a bit with, trying to figure out which one is the right choice for each use case. I think there's some good info in [this thread](https://twitter.com/kentcdodds/status/1351662112200822785).

I tend to do the same as what someone else said in that thread: use interfaces when I can and types when I can't.